config.js

```diff
@@ -41,7 +41,8 @@ module.exports = require('yargs')
         'mqtt-url': 'mqtt://127.0.0.1',
         name: 'hm',
         verbosity: 'info',
-        'listen-address': require('./firstip.js'),
+        'listen-address': '0.0.0.0',
```

Comment:
if we set init-address to firstip by default we can also set listen-address to firstip. Or we leave listen-adress on 0.0.0.0 but then it makes no sense to set a init-address default.

Reply:
That's a Docker-related change: Docker may have multiple (virtual) network interfaces, so if you bind 0.0.0.0 by default you need to configure only `--init-address`, otherwise both `--init-address` and `--listen-address`. It's more like a convenience for Docker users to set the defaults this way. I'm fine with both as long as it's documented somewhere.

Reply:
I see, got the point. This change would it make more convenient for you (only 1 option has to be set) but less convenient for me (I would need to set an extra option because I don't want it to listen on all interfaces). Will think about it. Perhaps just toss a coin. U want head or tail? ;-)